nvm.sh

```diff
@@ -1485,7 +1502,7 @@ nvm() {
       if [ "_$NVM_OS" = "_freebsd" ]; then
         # node.js and io.js do not have a FreeBSD binary
         nobinary=1
-      elif [ "_$NVM_OS" = "_sunos" ] && ([ "$NVM_IOJS" = true ] || [ "$NVM_NODE_MERGED" = true ]); then
+      elif [ "_$NVM_OS" = "_sunos" ] && [ "$NVM_IOJS" = true ]; then
```

Comment:
It looks like https://iojs.org/dist/v3.3.1/ is the only `io.js` version that has a `sunos` binary - could we change this to only set nobinary to 1 if the version is less than `v3.3.1`?